modules/odr_core/odr_core/crud/content.py

```diff
@@ -59,21 +151,42 @@ def update_content(db: Session, content_id: int, content: ContentUpdate) -> Opti
 def delete_content(db: Session, content_id: int) -> bool:
     db_content = db.query(Content).filter(Content.id == content_id).first()
     if db_content:
+        # Delete associated sources
+        for source in get_content_sources(db, content_id):
```

Comment:
I would use ondelete='CASCADE' / cascade='all,delete-orphan' instead of removing it in loop

Reply:
good put